src/reducers/authReducer.js

```diff
@@ -0,0 +1,25 @@
+import types from '../actions/actionTypes';
+
+export const initialState = {
+  isLoading: false,
+  errors: {
+    status: false,
+    message: '',
+    response: {},
+  },
+  user: {},
```

Comment:
Since there is going to be a profile reducer, this user belongs there. So you can remove it.

Reply:
Not sure I understand what you mean. Pls can you explain further for clarity? Thanks

Reply:
Godfry is working on the profile, so he will intercept all login success or signup success actions in a redux-saga and dispatch an action with the `user` returned on login as the payload, and save it in a `profile` reducer. All you need to do is remove the `user` from your reducer.